lexicons/app/bsky/feed/getAuthorFeed.json

```diff
@@ -11,7 +11,8 @@
         "properties": {
           "actor": {"type": "string", "format": "at-identifier"},
           "limit": {"type": "integer", "minimum": 1, "maximum": 100, "default": 50},
-          "cursor": {"type": "string"}
+          "cursor": {"type": "string"},
+          "filter": {"type": "string", "enum": ["posts_and_replies", "posts", "media"], "default": "posts_and_replies"}
```

Comment:
I think I'd do `knownValues` here instead of `enum` since it allows us to add more going forward. An `enum` can never change :/

Reply:
Roger roger